Mathlib/Algebra/Group/Defs.lean

```diff
@@ -43,6 +43,24 @@ actions and register the following instances:
 
 open Function
 
+/--
+The notation typeclass for heterogeneous scalar multiplication.
+This enables the notation `a • b : γ` where `a : α`, `b : β`.
+-/
```

Comment:
It looks like this docstring was overlooked

Reply:
Fixed in #711